_posts/blog/2020-01-23-2020-01-23-lab-meeting.md

```diff
@@ -0,0 +1,173 @@
+* Introducing speakers with a sentence or two if you know them well enough.
+* Asking how to pronounce speaker names _before_ the session!
+
+**Yo** asked a question that we discussed in detail: how do we each feel about absentee managers / leaders / mentors?
+
+Some of the lab's comments and experiences included:
+
+* I used to think "rarely available but nice when around" was fine, but I feel like making an active effort to be hands-on and not always busy is actually pretty important if you want people to feel valued.
+* Preparing briefing docs or a clear powerpoint on my work for an overly busy supervisor really helps to get useful input when we did meet.
+* Praise is so important, but it has to be real.
+  Hearing "well done" if there isn't enough trust to believe that my manager actually means it rings hollow.
+* One thing I try to do when, say, reviewing a document, is to highlight bits and explain _why_ I liked them, to make the praise seem real-er.
+* Having a busy supervisor nudged me to learn to communicate with my colleagues more often.
+* It's particularly difficult to know how to make a decision before joining a position.
+  The job or doctoral project description is so short, and an interview won't give you information on whether your mentor is good or not.
+* A major challenge is that these are structural problems in academia.
+  Spending time supporting students and staff on anything that isn't a co-authored paper is actively disincentivsed by academic reward structure.
+* A good phd supervisor at the start of your degree is a bad one at the end, and vice versa.
+  Its important not to look for everything you need in one person.
+* The time scale of impact from mentorship and training is long.
+  The [mathematics geneology project](https://genealogy.math.ndsu.nodak.edu) and [The Academic Family Tree](https://academictree.org) show if people train others who go on to be successful... in academia only.
+  These projects don't track success if trainees leave academia 😭
+* Professional services can support you.....but they're often not promoted or enabled.
+* Don't look for a specific mentor, build a mentor from LOTS of different people and their strengths.
+* If the goals of supervisor and student are not aligned then they could try to design outputs that are going to benefit both.
+  For example if the student wants to go into industry, they could focus on building infrastructure and pipelines in the lab rather than writing papers.
+  (Although this can backfire when it comes to the degree requirements - at least at the moment!)
+
+**Sarah** asked how to encourage people to use a service?
+Hub23 (the Turing's private BinderHub) doesn't currently have any users which is a shame because she's put a lot of work into making it really easy for me to maintain (e.g. [writing bots](https://github.com/HelmUpgradeBot) to keep the helm chart up-to-date and the container registry in a manageable state).
```

Comment:
```suggestion
Hub23 (the Turing's private BinderHub) doesn't currently have any users which is a shame because she's put a lot of work into making it really easy to maintain (e.g. [writing bots](https://github.com/HelmUpgradeBot) to keep the helm chart up-to-date and the container registry in a manageable state).
```